### PR TITLE
Table & TableCell Components: Remove Table margin and add TableCell alignment prop

### PR DESCRIPTION
**Pull request by natalynyu**
This PR removes the margin around the `Table` component, since the margin was causing a horizontal scrollbar to appear in Course Planning. The margin was moved to the `PageBody` component instead. Also, the prop `alignment` was added to the `TableCell` component so that users can adjust how the content within the `td` is positioned (`text-align` is set to `left`, `right`, or `center`). The default value of the `alignment` prop is `left`.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality
      to change)

## Checklist:

- [x] I have run `eslint` on the code
- [x] I have added JSDoc for all of my code (where applicable)
- [ ] I have added tests to cover my changes.

## Priority:
- [x] Normal 
- [ ] High 

## Related Issues:

Addresses [#117](https://github.com/seas-computing/course-planner/issues/117)

### CODE REVIEW

File: src/Tables/TableCell.tsx [hunk 1]
```diff
@@ -1,30 +1,49 @@
...
-  FunctionComponent, ReactElement, ReactNode, useContext,
+  FunctionComponent,
+  ReactElement,
+  ReactNode,
+  useContext,
 } from 'react';
 import styled, { ThemeContext } from 'styled-components';
 import { BaseTheme } from '../Theme';
 
+export enum ALIGN {
```
Comment (rmainseas) [inline src/Tables/TableCell.tsx]: This could probably do with some jsdoc documentation on it

File: src/Tables/__tests__/Table.test.tsx [hunk 2]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { render } from 'test-utils';
 import convert from 'color-convert';
 import { strictEqual } from 'assert';
-import TableCell from '../TableCell';
+import TableCell, { ALIGN } from '../TableCell';
```
Comment (rmainseas) [inline src/Tables/__tests__/Table.test.tsx]: I would add some tests in here to make sure that:

- `ALIGN.LEFT`, `ALIGN.CENTER` and `ALIGN.RIGHT` do what you expect
- The default value is `ALIGN.LEFT`

Comment (natalynyu) [reply src/Tables/__tests__/Table.test.tsx]: Added tests through 5619e56

File: src/Tables/TableCell.tsx (hunk 1 above)
Comment (natalynyu) [reply src/Tables/TableCell.tsx]: Added jsdoc comment through 7434ce7

**Pull request merged**